templates/wechatgame/first-screen.ejs

```diff
@@ -179,9 +179,10 @@ function initProgressVertexBuffer() {
 }
 
 function updateVertexBuffer() {
-    // the ratio value may be adjusted according to the expected image display effect
-    const widthRatio = image.width / canvas.width * 1.35 * displayRatio; 
-    const heightRatio = image.height / canvas.height * 1.35 * displayRatio;
+    // By default, maintain aspect ratio by constraining display at 200px height
+    const defaultRatio = 200 / image.width;
```

Comment:
Why use 200? It seems a magic number.

Reply:
1. cocos default logo is 144 * 200
2. defualt width in splashScreen.ts is also 200
![image](https://github.com/cocos/cocos-engine/assets/18634782/7fca385a-6558-4ea7-acb9-ba4726a8de82)